Backend/Middleware/src/main/java/middleware/model/User.java

```diff
@@ -0,0 +1,44 @@
+package middleware.model;
+
+import jakarta.persistence.*;
+import lombok.Data;
+
+import java.util.Date;
+
+/**
+ * Represents a user entity in the system.
+ * This entity is mapped to the "User" table in the database.
```

Comment:
this is the Jakarta Bean ORM thing issit

Reply:
Yeah so each java entity is mapped to the MySQL DB "UserMSVC" table "User"

Reply:
Do uw me to add more to the comments

Reply:
Changed for JWToken also